### ISSUE TITLE
is filter grok still available now in puppet-logstash

### ISSUE THREAD

**Issue by mizeng** [elastic/puppet-logstash#278]
I've find this in readme.md:
```
grok {
      patterns_dir => "/etc/logstash/patterns/"
      ...
}
```

But I can't find the grok class in puppet-logstash. So I wonder is it still available?

**Comment by mizeng** [comment]
or above statements are only content rather then puppet code which is located in /etc/logstash/conf.d/logstash.conf?

**Comment by jarpy** [comment]
Correct :)

That's Logstash code, not Puppet code, but they look very similar because they both use Ruby-like hash syntax.

I'll try to make that more apparent in the README.

**Comment by jarpy** [comment]
Fixed in d7362846f682d5c92c79f3bbee6882d7ebef3156.
